### DISCUSSION TITLE
How can I select the actions I subscribe to with the realtime api

### DISCUSSION THREAD

**Discussion by ChrisPoul** [pocketbase/pocketbase#3826] [Q&A]
I am using the `subscribe` and `unsubscribe` methods and but I don't want to subscribe to all types of action, just specific ones like `create` or `update` .

Is this possible?

**Comment by ganigeorgiev** [comment]
No. The topics are usually either the wildcard (`*`) for the collection or a single record id, but you can't subscribe only to specific actions.

At the moment if you want to filter the actions you can do it on the client-side from within the callback.

```js
pb.collection("example").subscribe("*", (e) => {
    if (e.action != "delete") {
        return;
    }
    
    // handle delete only ...
})
```
